pkg/services/serviceaccounts/api/api.go

```diff
@@ -229,45 +207,26 @@ func (api *ServiceAccountsAPI) SearchOrgServiceAccountsWithPaging(c *models.ReqC
 	if page < 1 {
 		page = 1
 	}
-	query := &models.SearchOrgUsersQuery{
-		OrgID:            c.OrgId,
-		Query:            c.Query("query"),
-		Page:             page,
-		Limit:            perPage,
-		User:             c.SignedInUser,
-		IsServiceAccount: true,
-	}
-	serviceAccounts, err := api.store.SearchOrgServiceAccounts(ctx, query)
+	serviceAccountSearch, err := api.store.SearchOrgServiceAccounts(ctx, c.OrgId, c.Query("query"), page, perPage, c.SignedInUser)
```

Comment:
i really like this approach better!

Reply:
this is the way going forward right?

Reply:
@eleijonmarck from what I've seen yes, it avoids hiding mandatory args in structures